Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = ["crates/op/*", "crates/psh-system"]
 exclude = ["test_resources/profiling"]
```

Comment:
The test_resources dir not the member of the workspace. I don't think they need to be modified.

Reply:
Yes, but it's better to keep all packages at the same version.